Doc/library/venv.rst

```diff
@@ -15,12 +15,22 @@
 
 --------------
 
+.. _venv-def:
+
 The :mod:`venv` module provides support for creating lightweight "virtual
-environments" with their own site directories, optionally isolated from system
-site directories.  Each virtual environment has its own Python binary (which
-matches the version of the binary that was used to create this environment) and
-can have its own independent set of installed Python packages in its site
-directories.
+environments" with their own independent set of Python packages installed in
+their site directories. A virtual environment is created on top of an existing
+Python installation, known as the virtual environment's "base" Python, and may
+be optionally isolated from the base's site packages, meaning
+that packages installed in the base environment will not be accessible from
+the virtual environment.
+A virtual environment is therefore a powerful and convenient concept which
```

Comment:
Perhaps insert a blank line here?

Reply:
Sure thing. Done in [1ba4797](https://github.com/python/cpython/pull/98350/commits/1ba4797ad0149e3e1c27eb6bf0bf5b9d57256ff7)